client/vscode/test/fixtures/workspace/.vscode/settings.json

```diff
@@ -6,6 +6,9 @@
     // "https://sourcegraph.test:3443/.api/opencodegraph": true,
     // "../../../../../../provider/hello-world/dist/index.js": true,
     "https://opencodegraph.org/npm/@opencodegraph/provider-hello-world": true,
+    "https://opencodegraph.org/npm/@opencodegraph/provider-techstack": {
```

Comment:
@sqs can you help me host this provider at opencodegraph.org? how can I test the provider on VSCode locally?

Reply:
opencodegraph.org/npm/... proxies any npm module. So it should already work!